src/components/common/modal/folderNameInput/folderNameInput.tsx

```diff
@@ -1,9 +1,9 @@
-import { useState } from 'react';
+import { ChangeEvent, useState } from 'react';
 import styles from './FolderNameInput.module.css';
 
 const FolderNameInput = ({ folderName = '' }) => {
   const [value, setValue] = useState(folderName);
-  const handleChange = e => setValue(e.target.value);
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => setValue(e.target.value);

```

Comment:
```suggestion
  const handleChange: ChangeEventHandler<HTMLInputElement> = (e) => setValue(e.target.value);

```

함수 표현식과 타입이 호환성이 좋다는 표현을 자주하는데 이렇게 변수에 바로 함수 타입을 지정해줄 수 있기 때문입니다, 이렇게 관리하면 파라미터를 깔끔하게 관리할 수 있습니다.